src/styles/layout/_component-grid.scss

```diff
@@ -111,42 +113,6 @@ $breakpoint-accordion-large-755: 47rem;
           bottom: -12rem;
```

Comment:
Have you experimented with using `top` rather than `bottom` at all? I'm wondering if you set `top` to some value if it might stay more consistent as the screen is being resized, making it possible to avoid adding some of these breakpoints here that only serve to adjust `bottom`. You'd have to adjust `object-position` to `top` for the accordion img as well, something like this:
![image](https://user-images.githubusercontent.com/54716846/204983642-cb6ebdc6-7158-47c6-9f40-00e8d397fb02.png)

There might be something about this approach that totally doesn't work that I missed, so I'm curious to know your thoughts.

Reply:
I did mess around with `top`, but maybe I wasn't putting` top` on just the accordion image. Because Accordion is the only illustration that is cropped differently, I wonder if it's worth it to edit the accordion image, get rid of a ton of the padding and white space, crop it to match what I'm trying to do here, and then use some sort of `picture` and `source` combination like we have for the hero image?

Reply:
In this case, I would check that CSS can do the job first, it may not be worthwhile to have a cropped image that you switch out

Reply:
How about this: https://github.com/sparkbox/accessible-components/pull/110/commits/349c925d4c76206ec981764a251703bd81f27551